Makefile

```diff
@@ -153,6 +154,11 @@ superuser: ## create a Django superuser
 	@$(MANAGE) createsuperuser
 .PHONY: superuser
 
+# -- Terraform
+
+env.d/terraform:
+	cp env.d/terraform.dist env.d/terraform
+
```

Comment:
In this project, terraform is a directory not a file. Then this folder should contain an `example.dist` file to create a new terraform project. So I don't think this command is useful.
```suggestion
```

Reply:
I have removed example.dist so let's keep this?

Reply:
Hmm.. we could remove this command as `terraform.dist` does not exist.

Reply:
ok and I should probably add a .gitkeep in /env.d/terraform to keep this folder for developpers to add their openrc.sh files. WDYT?

Reply:
Perfect 👌

Reply:
:+1: